Core/GameEngineDevice/Source/Win32Device/Common/Win32BIGFileSystem.cpp

```diff
@@ -216,6 +216,14 @@ Bool Win32BIGFileSystem::loadBigFilesFromDirectory(AsciiString dir, AsciiString
 	Bool actuallyAdded = FALSE;
 	FilenameListIter it = filenameList.begin();
 	while (it != filenameList.end()) {
+		AsciiString filePath = (*it);
+		filePath.toLower();
+		// TheSuperHackers @fix bobtista 18/11/2025 Skip duplicate INIZH.big in Data\INI to prevent CRC mismatches on EA App/Origin installs
+		if (strstr(filePath.str(), "data\\ini\\inizh.big") != NULL || strstr(filePath.str(), "data/ini/inizh.big") != NULL) {
```

Comment:
`"data\\ini\\inizh.big"` path will not work if `dir` is not game root. Anything we can do about that?

Reply:
Yes, `endsWithNoCase` handles this, thanks for pointing it out to me. It matches the path suffix, so it works whether we're loading from the root directory or from installPath - both paths ending with `data\ini\inizh.big` will be detected.